### PR TITLE
Bugfix of NPE in LongLine rule on Binary Expressions

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Fix bug with NPE: we didn't consider, that offset changes in course of fixes, so it's break the logic sometimes
* Add additional check for long comments: if there is only one white space, which is located after `//` - we shouldn't fix split it
* Add tests

### CODE REVIEW

**Comment by kgevorkyan** [comment]
Please take a look for a [explanation](https://github.com/cqfn/diKTat/issues/920#issuecomment-863198257), if changes are unclear

**Review by petertrr — APPROVED** [review]
LGTM, sounds like it was a really tricky bug

File: diktat-rules/src/test/resources/test/paragraph3/long_line/LongStringTemplateExpected.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+object Observables {
+    // Some
+// looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong line
```
Comment (akuleshov7) [inline diktat-rules/src/test/resources/test/paragraph3/long_line/LongStringTemplateExpected.kt:3]: need a following test also :
val someCode = 15 //loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongggggggggggggggggggggggggg

**Review by akuleshov7 — APPROVED** [review]

File: diktat-rules/src/test/resources/test/paragraph3/long_line/LongStringTemplateExpected.kt (hunk 1 above)
Comment (kgevorkyan) [reply diktat-rules/src/test/resources/test/paragraph3/long_line/LongStringTemplateExpected.kt:4]: Added

**Pull request merged**